refinery/factory_boy/django_model_factories.py

```diff
@@ -16,6 +16,7 @@ class Meta:
     name = "Test DataSet - {}".format(uuid)
     creation_date = datetime.now()
     modification_date = datetime.now()
+    slug = None
```

Comment:
I think you could already use the factory here to create a Dataset with a `slug` without adding this line.
I could also very well be wrong and would like to know if I am!